schema/schema.json

```diff
@@ -1227,7 +1227,9 @@
       "type": "string"
     },
     "DateTime": {
+      "description": "Regex-based implementation of iso-date-time until available in ajv-formats@3.0.0",
       "format": "date-time",
+      "pattern": "^[0-9]{4}-((0[13578]|1[02])-(0[1-9]|[12][0-9]|3[01])|(0[469]|11)-(0[1-9]|[12][0-9]|30)|(02)-(0[1-9]|[12][0-9]))T(0[0-9]|1[0-9]|2[0-3]):(0[0-9]|[1-5][0-9]):(0[0-9]|[1-5][0-9])\\.[0-9]{3}Z$",
```

Comment:
This regex is too strict now - according to [the RFC 3339 spec][1] the second fraction is optional and is any number of digits. Also the `T` and `Z` may be upper/lower case and timezone offsets are allowed, e.g. `2023-08-31T00:00:00+01:00`.

[1]: https://datatracker.ietf.org/doc/html/rfc3339#section-5.6

Reply:
Thanks for catching, examples updated to catch these more diverse scenarios and [regex sourced from here](https://gist.github.com/marcelotmelo/b67f58a08bee6c2468f8) now